modules/nf-core/blast/blastp/main.nf

```diff
@@ -0,0 +1,57 @@
+process BLAST_BLASTP {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::blast=2.14.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/blast:2.14.1--pl5321h6f7f691_0':
+        'biocontainers/blast:2.14.1--pl5321h6f7f691_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    path(db)
+
+    output:
+    tuple val(meta), path("*.blastp.csv"), emit: csv
+    path "versions.yml"                  , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def is_compressed = fasta.name.endsWith(".gz")
+    def fasta_name = fasta.name.replace(".gz", "")
+    """
+    if [ "$is_compressed" == "true" ]; then
+        gzip -c -d $fasta > $fasta_name
+    fi
+
+    DB=`find -L ./ -name "*.phr" | sed 's/\\.phr\$//'`
+    blastp \\
+        -db \$DB \\
+        -query ${fasta_name} \\
+        -out ${prefix}.blastp.csv \\
```

Comment:
```suggestion
        -out ${prefix}.csv \\
```
It's preferred to leave out custom extensions. You can still add them yourself with the prefix without enforcing this on every user (and it gives greater flexibility which is always nice :) )

Reply:
Noted and updated!